spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -1095,9 +1095,9 @@ def write_units(sorting: se.SortingExtractor, nwbfile,
                 warnings.warn(f"Description for property {pr} not found in property_descriptions. "
                               f"Setting description to 'no description'")
 
+        skip_properties = ["mda_max_channel"]
```

Comment:
This should be an arg of the function

Reply:
Easy to do; do you think in this case `mda_max_channel` should be by default ignored or add it in as a specified ignore in the pipelines where it's relevant?

Reply:
@CodyCBakerPhD @bendichter I think that the `mda_max_channel` is loaded in the `MdaSortingExtractor`. Maybe the best option for now would be to make it optional and have ironclust not loading it by default.

However, we should have a `skip_properties` and `skip_features` as extra arguments